deploy/crds/applicationmonitoring.integreatly.org_applicationmonitorings_crd.yaml

```diff
@@ -10,6 +10,8 @@ spec:
     plural: applicationmonitorings
     singular: applicationmonitoring
   scope: Namespaced
+  subresources:
```

Comment:
This needs to be removed or otherwise the Operator will get stuck unable to update it's status.
Alternatively we can update to `client.Status().Update(...)` here: https://github.com/integr8ly/application-monitoring-operator/blob/master/pkg/controller/applicationmonitoring/applicationmonitoring_controller.go#L762

Reply:
Agreed. I recreated the crd's before the final commit, so it made it's way back in.